Src/AmrCore/AMReX_Interpolater.H

```diff
@@ -57,6 +57,39 @@ public:
                          int              actual_state,
                          RunOn            gpu_or_cpu) = 0;
 
+    /**
+    * \brief Coarse to fine interpolation in space for face-based data.
+    *
+    * \param crse
+    * \param crse_comp
+    * \param fine
+    * \param fine_comp
+    * \param ncomp
+    * \param fine_region
+    * \param fine_values
+    * \param fine_known
+    * \param ratio
+    * \param crse_geom
+    * \param fine_geom
+    * \param bcr
+    * \param bccomp
+    * \param runon
+    */
+    virtual void interp_face (const FArrayBox& /*crse*/,
+                              const int        /*crse_comp*/,
+                              FArrayBox&       /*fine*/,
+                              const int        /*fine_comp*/,
+                              const int        /*ncomp*/,
+                              const Box&       /*fine_region*/,
+                              const IntVect&   /*ratio*/,
+                              IArrayBox* const& /*solve_mask*/,
```

Comment:
Let's change this (and overrides) from `IArrayBox* const&` to `IArratBox const*`.   The problem of `IArrayBox* const& solve_mask` is it actually allows for `solve_mask->setVal(0)`.  So it's the wrong kind of `const.

Reply:
Maybe a better option would be `const IArrayBox&` ? For this single component case, `solve_mask` doesn't need to be a pointer at all.

Reply:
Yes, agreed.